Upstream-Status: Pending 

Automake 1.12 has deprecated automatic de-ANSI-fication support

Fix this issue with automake 1.12:

| configure.ac:23: error: automatic de-ANSI-fication support has been removed

Signed-Off-by: Nitin A Kamble <nitin.a.kamble@intel.com>
2012/05/02

Index: git/configure.ac
===================================================================
--- git.orig/configure.ac
+++ git/configure.ac
@@ -20,7 +20,6 @@ AC_PROG_CXX
 AC_PROG_CPP
 AM_PROG_CC_STDC
 AM_PROG_CC_C_O
-AM_C_PROTOTYPES
 AC_PROG_RANLIB
 AC_OBJEXT
 AC_EXEEXT
